doc/main.md

```diff
@@ -13,6 +13,10 @@ The extension can function in one of two modes:
 
 Each mode is described in detail on its own page.
 
+Furthermore, the extension provides:
+
+* [Linting (Showing diagnostics in the active text editor)](linting.md)

```

Comment:
I assume you are describing what linting is, and it is: "the showing of diagnostics in the active text editor".